lte/gateway/c/core/oai/lib/itti/intertask_interface.c

```diff
@@ -37,6 +37,7 @@
 #include <string.h>
 #include <malloc.h>
 #include <stdint.h>
+#include <liblfds710.h>
```

Comment:
Just curious, which function is this dependency for?

Reply:
(@lionelgo)

Reply:
This is for this macro : LFDS710_MISC_MAKE_VALID_ON_CURRENT_LOGICAL_CORE_INITS_COMPLETED_BEFORE_NOW_ON_ANY_OTHER_LOGICAL_CORE. (But may be we don't need this lib anymore)

Reply:
Ah OK. So maybe we were pulling in the lfds header file somewhere up in the dependency before this change? Could you elaborate on why we might not need it anymore?

Reply:
We can still see some references to this lib in [lte/gateway/c/core/oai/common/shared_ts_log.h ](https://github.com/magma/magma/blob/58d67382cf011d82576ff5884c82c7a5e9dea3ab/lte/gateway/c/core/oai/common/shared_ts_log.h#L62)